inc/saiobject.h

```diff
@@ -249,6 +249,30 @@ sai_status_t sai_query_stats_capability(
         _In_ sai_object_type_t object_type,
         _Inout_ sai_stat_capability_list_t *stats_capability);
 
+/**
+ * @brief Bulk objects get statistics.
+ *
+ * @param[in] switch_id SAI Switch object id
+ * @param[in] object_type Object type
+ * @param[in] object_count Number of objects to get the stats
+ * @param[in] object_key List of object keys
+ * @param[in] number_of_counters Number of counters in the array
+ * @param[in] counter_ids Specifies the array of counter ids
+ * @param[in] mode Statistics mode
+ * @param[out] counters Array of resulting counter values
+ *
+ * @return #SAI_STATUS_SUCCESS on success, failure status code on error
```

Comment:
* @return #SAI_STATUS_SUCCESS on success when all objects stats get succeeds.
 * #SAI_STATUS_FAILURE when any of the get stats fails. When there is
 * failure, Caller is expected to go through the list of returned statuses to
 * find out which fails and which succeeds.

Reply:
https://github.com/opencomputeproject/SAI/pull/1352/commits/9a071860458785a2f15e52b512c154bef6b691d3